packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -26,10 +26,10 @@ export interface IBasePickerState {
   selectedIndices?: number[];
 }
 
-export const enum PickerAriaIdEnum {
-  selectedSuggestionAlert,
-  selectedItems,
-  suggestionList
+export interface IPickerAriaIds {
```

Comment:
Would a type literal be more appropriate, given that as far as I can tell this is internally defined and not passed in? 

Also since you're exporting it, please add JSDoc :) To the properties but also to the interface or type